unity-renderer/Assets/Scripts/MainScripts/DCL/WebInterface/Interface.cs

```diff
@@ -609,7 +609,7 @@ public class SearchENSOwnerPayload
             public int maxResults;
         }
 
-#if UNITY_WEBGL && !UNITY_EDITOR
+#if UNITY_WEBGL && !(UNITY_EDITOR || UNITY_STANDALONE)
```

Comment:
Same as prev comments. Build can't have `UNITY_STANDALONE` and `UNITY_WEBGL` at the same time.

Reply:
And it can have UNITY_EDITOR and UNITY_WEBGL at the same time? :thinking: